OEBPS/Chapter1.xhtml

```diff
@@ -1,176 +1,176 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version='1.0' encoding='utf-8'?>
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
-    <meta charset="utf-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```

Comment:
Huh, I'm a little surprised you're allowed to mix MIME types like this. But ePubCheck and [the XHTML spec](https://www.w3.org/TR/xhtml1/#C_9) both say it's fine. 🤷‍♀️